ugsync/src/main/java/org/apache/ranger/unixusersync/process/UnixUserGroupBuilder.java

```diff
@@ -538,7 +538,7 @@ private void buildUnixGroupList(String allGroupsCmd, String groupCmd, boolean us
 
 			for (String group : groups) {
 				String command = String.format(groupCmd, group);
-				String[] cmd = new String[]{"bash", "-c", command + " '" + group + "'"};
+				String[] cmd = new String[]{"bash", "-c", command};
```

Comment:
It looks like this fix would be needed in line #507 as well. @fateh288  - can you please review?

Reply:
Thanks @mneethiraj 
Created a new pull request for this with the suggested changes.